### PR TITLE
Vishal/137 updated pr template

### PR DESCRIPTION
**Pull request by vishal-codes**
Resolves: #134 

## How to Test
Check out the `pull_request_template.md` file

## Potential Side Effects/Things to Look Out For
NO Potential Side Effects

### CODE REVIEW

File: changelogs/2021-09-10T20-11-51.999+05-30.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[NEW] A user-friendly description of your change {#137}
```
Comment (mrjones2014) [inline changelogs/2021-09-10T20-11-51.999+05-30.md:1]: This should be `[IMPROVED]` and provide a description of your change -- this is the default placeholder changelog.

**Review by mrjones2014 — CHANGES_REQUESTED** [review]

File: changelogs/2021-09-10T20-11-51.999+05-30.md (hunk 1 above)
Comment (vishal-codes) [reply changelogs/2021-09-10T20-11-51.999+05-30.md:1]: ohh okaay

Comment (vishal-codes) [reply changelogs/2021-09-10T20-11-51.999+05-30.md:1]: Done  !! @mrjones2014

File: changelogs/2021-09-10T20-11-51.999+05-30.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Upadated PR template to make sure people are reading the contributing guidelines. Made changes in the pull_request_template.md file.
```
Comment (mrjones2014) [inline changelogs/2021-09-10T20-11-51.999+05-30.md:2]: I meant like this:

```suggestion
[IMPROVED] Upadated PR template to make sure people are reading the contributing guidelines. {#137}
```

Please read through the `README.md`, `COMMANDS.md`, and `CONTRIBUTING.md` to get an understanding of how the CLI works.

Comment (mrjones2014) [reply changelogs/2021-09-10T20-11-51.999+05-30.md]: I've added it for you this time.

**Review by mrjones2014 — APPROVED** [review]

**Comment by vishal-codes** [comment]
@mrjones2014 should I delete the branch?

**Comment by mrjones2014** [comment]
@vishal-codes your choice. Your branch only exists in your fork. The PR was merged into master in this repo.

**Comment by mrjones2014** [comment]
@vishal-codes please consider starring the repo if you like it :)

**Comment by mrjones2014** [comment]
@vishal-codes please make sure you link the correct issue next time. This had unintentionally closed an issue that this PR did not resolve.

**Pull request merged**